docs/advanced/theme.story.mdx

```diff
@@ -156,11 +157,10 @@ Use spacings for gutters, margins, and paddings. Don't use it for border width,
 ### Icon sizes
 
 ```
-theme.iconSizes.[byte|kilo|mega|giga]
+theme.iconSizes.[kilo|mega|giga]
```

Comment:
Was this a mistake in the docs? Or did we remove this icon size at some point? (similar questions for the changes below in this file)

Reply:
We're removing it with v2 and the release of the @sumup/icons package. 

The `byte` size (14px) should be replaced by the `kilo` (16px) size. A quick GitHub search showed that the `byte` size is barely used anyway.

I'm planning to release a minor version of Circuit UI v1 that will have deprecation warnings for all breaking changes to make the upgrade easier.